.github/workflows/template/build-{{ostype}}.yaml.j2

```
@@ -54,7 +54,17 @@ jobs:
           bash <(wget -q -O- http://neuro.debian.net/_files/neurodebian-travis.sh)
           sudo apt-get update -qq
           sudo apt-get install eatmydata
-          sudo eatmydata apt-get install singularity-container gnupg moreutils strace
+          sudo eatmydata apt-get install gnupg moreutils strace
+
+      - name: Install Singularity
+        run: |
+          release="$(curl -fsSL https://api.github.com/repos/sylabs/singularity/releases/latest | jq -r .tag_name)"
```

since the goal is not to test singularity here , so to remove possible variance, let's may be just
```suggestion
          # use if decide to get most recent release of singularity
          # release="$(curl -fsSL https://api.github.com/repos/sylabs/singularity/releases/latest | jq -r .tag_name)"
          release="v3.10.4"
```
if that is the one which works for us ATM?